webview/src/experiments/components/App.test.tsx

```diff
@@ -1215,7 +1216,7 @@ describe('App', () => {
       expect(disabledMenuItem).toBeDefined()
 
       disabledMenuItem && fireEvent.click(disabledMenuItem, { bubbles: true })
-      expect(screen.queryAllByRole('menuitem')).toHaveLength(13)
+      expect(screen.queryAllByRole('menuitem')).toHaveLength(14)
```

Comment:
You weren't the one to introduce the repeating of menu item length, but it might be a good idea to make it a variable to be reused. Next time, the number would have to be changed once only.